app/component/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactNode } from "react";
+import * as styles from "./styles.css";
+
+export function Header(): ReactNode {
+  return (
+    <header className={styles.header}>
+      <div className={styles.headerMenu}>
```

Comment:
```suggestion
      <nav className={styles.headerMenu}>
```
navの方がいいかも？
https://developer.mozilla.org/ja/docs/Web/HTML/Element/nav